docs/README.md

```diff
@@ -0,0 +1 @@
+Let's use this directory to store all relevant documentation in .md files.
```

Comment:
Since this file will be read by anyone, I'd prefer to remove the "Let's use", same as the readme for the src folder.

Suggestion:
>This directory stores all relevant documentation in .md files.